Scan/utkscan/analyzers/source/Gsl2Fitter.cpp

```diff
@@ -89,21 +83,20 @@ void GslFitter::PerformFit(const std::vector<double> &data,
     }
 
     gsl_multifit_fdfsolver_wset (s, &f, &x.vector, &w.vector);
-    gsl_multifit_fdfsolver_driver(s, 1000, xtol, gtol, ftol, &info);
+    gsl_multifit_fdfsolver_driver(s, 100, xtol, gtol, ftol, &info);
```

Comment:
That number is the maximum number of iterations, the major change is reducing the tolerance from 10^-10 to 10^-4.